CHANGELOG

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Fuse.js search feature
```

Comment:
More accurately, we added customized search feature for `DataTable`. But we can mention that `fuse.js` based implementation is in the storybook.